infra/Pos/Network/Yaml.hs

```diff
@@ -228,72 +222,3 @@ instance FromJSON Topology where
 
 instance IsConfig Topology where
   configPrefix = return Nothing
-
-{-------------------------------------------------------------------------------
-  ToJSON instances
--------------------------------------------------------------------------------}
-
-instance ToJSON NodeRegion where
-  toJSON (NodeRegion region) = toJSON region
-
-instance ToJSON NodeName where
-  toJSON (NodeName name) = toJSON name
-
-instance ToJSON NodeRoutes where
-  toJSON (NodeRoutes routes) = toJSON routes
-
-instance ToJSON NodeType where
-  toJSON NodeCore  = toJSON ("core"  :: Text)
-  toJSON NodeEdge  = toJSON ("edge"  :: Text)
-  toJSON NodeRelay = toJSON ("relay" :: Text)
-
-instance ToJSON DnsDomains where
-  toJSON DnsDomains{..} = toJSON $ aux dnsDomains
-    where
-      aux :: [[DNS.Domain]] -> [[String]]
-      aux = map (map BS.C8.unpack)
-
-instance ToJSON NodeMetadata where
-  toJSON NodeMetadata{..} = A.object $ addAddress nmAddress [
-        "type"          .= nmType
-      , "region"        .= nmRegion
-      , "static-routes" .= nmRoutes
-      ]
-    where
-      addAddress :: NodeAddr -> [A.Pair] -> [A.Pair]
-      addAddress (NodeAddrExact addr mPort) = (++) $ concat [
-          [ "host" .= aux addr ]
-        , [ "port" .= p | Just p <- [mPort] ]
-        ]
-      addAddress (NodeAddrDNS mHost mPort) = (++) $ concat [
-          [ "host" .= aux h | Just h <- [mHost] ]
-        , [ "port" .= p     | Just p <- [mPort] ]
-        ]
-
-      aux :: DNS.Domain -> String
-      aux = BS.C8.unpack
-
-instance ToJSON AllStaticallyKnownPeers where
-  toJSON AllStaticallyKnownPeers{..} =
-      A.object (map aux $ M.toList allStaticallyKnownPeers)
-    where
-      aux :: (NodeName, NodeMetadata) -> A.Pair
-      aux (NodeName name, info) = name .= info
-
-instance ToJSON Topology where
-  toJSON (TopologyStatic    nodes)  = A.object [ "nodes"  .= nodes                   ]
-  toJSON (TopologyBehindNAT relays) = A.object [ "relays" .= relays                  ]
-  toJSON (TopologyP2P v f)          = A.object
-      [ "p2p" .= A.object [
-            "variant"   .= ("normal" :: Text)
-          , "valency"   .= v
-          , "fallbacks" .= f
-          ]
-      ]
-  toJSON (TopologyTraditional v f)  = A.object
-      [ "p2p" .= A.object [
-            "variant"   .= ("traditional" :: Text)
-          , "valency"   .= v
-          , "fallbacks" .= f
-          ]
-      ]
```

Comment:
We weren't using them for anything, so rather than keeping them updated and in sync, I just removed them. We can revive them if you think it's better.